packet/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -28,6 +28,12 @@ variable "type" {
   description = "Packet instance type for workers, can be changed afterwards to recreate the nodes"
 }
 
+variable "worker_clc_snippets" {
```

Comment:
What about using just clc_snippets, to match the variable name on AWS? https://github.com/kinvolk/lokomotive-kubernetes/blob/17043dcb736c070df1cb136a1f84ddb6af378cff/aws/flatcar-linux/kubernetes/workers/variables.tf#L73-L77